File: src/theme/colors.ts
// ORBII, Warm Editorial design system (v32).
//
// The app now shares the website's palette, so orbii.in and the phone in your
// hand look like the same company. That is the whole point of the change: a
// violet-on-white app is indistinguishable from every family-tracker on the
// store, and we were starting to look like a copy of one.
//
//   • CREAM   (#FAF9EC), the ground. Warm ivory, never grey, never white.
//   • INK     (#17161C), near-black text and dark editorial bands.
//   • LAVENDER(#8672CE), the one accent. CTAs, "protected", links, emphasis.
//   • CORAL   (#EF605E), SOS / danger only. Never used for anything calm.
//   • GOLD    (#D6A64F), premium (ORBII Plus) + highlights.
//   • SAGE    (#4FA383), "all clear" / verified confirmations.
//
// Every screen reads these tokens, so retinting here restyles the whole app.
// Nothing brand-related is hardcoded in screens, always reference a token.

const LAV = '#8672CE';
const LAV_DEEP = '#6E58B6';
const LAV_SOFT = '#E8DBF7';

export const colors = {
  // ── Canvas (warm ivory cream, the website ground) ────────
  cream: '#FAF9EC', // app background
  creamDeep: '#F3F0DF', // pressed / alt surface / tint band
  surface: '#FFFFFF', // cards
  surfaceAlt: '#FFFFFF', // insets

  // ── Lavender (primary accent / CTA / safe) ───────────────
  peach: LAV, // legacy name, now the brand lavender (drives most CTAs)
  peachDeep: LAV_DEEP,
  peachSoft: LAV_SOFT,

  // ── Sage (success / safe / all-clear). Green is the one colour everyone
  // reads as "you're okay". Keeping the `sage*` names so every existing screen
  // picks this up untouched.
  sage: '#4FA383',
  sageDeep: '#3E8268',
  sageSoft: '#E6F1EC',

  // ── Coral (SOS / alarm / danger) ─────────────────────────
  coral: '#EF605E',
  coralDeep: '#D14A48',
  coralSoft: '#FBE3E2',

  // ── Lavender tints (Voice SOS / trusted network) ─────────
  lavender: '#A493DC',
  lavenderDeep: LAV_DEEP,
  lavenderSoft: '#EADFF6',

  // ── Gold (premium / highlights) ──────────────────────────
  gold: '#D6A64F',
  goldDeep: '#B4842F',
  goldSoft: '#F5EAD3',

  // ── Text (warm neutrals, so type sits on cream not on grey) ──
  textPrimary: '#17161C',
  textSecondary: '#6B6560',
  textMuted: '#9A948C',
  textInverse: '#FFFFFF',
  /** Cream type for use on ink bands. */
  textOnInk: '#F3F0E4',

  // ── Lines (warm, soft, never dark) ───────────────────────
  border: '#E9E2CF',
  divider: 'rgba(50,42,20,0.09)',
  inputBorder: '#E9E2CF',
  inputBackground: '#FAF9EC',
  overlay: 'rgba(23,22,28,0.45)',

  // ── Semantic ─────────────────────────────────────────────
  success: '#4FA383',
  warning: '#D6A64F',
  error: '#EF605E',

  // ── Legacy aliases (remapped so every screen updates) ────
  brand: LAV,
  brandSoft: LAV_SOFT,
  brandMid: '#C0B2E8',
  brandDeep: LAV_DEEP,
  primary: '#EF605E', // SOS / danger primary, stays coral
  primaryDeep: '#D14A48',
  secondary: '#FFFFFF',
  accent: LAV,
  dark: '#17161C',
  darkSoft: '#201F27',
  background: '#FAF9EC',
  sectionBand: '#F3F0DF',
  surfaceMuted: '#F5F2E4',
} as const;

export type ColorKey = keyof typeof colors;

File: orbii_flutter/lib/core/theme/colors.dart
import 'package:flutter/material.dart';

/// ORBII — Warm Greige design system (ported 1:1 from the RN `colors.ts`).
///
/// Calm, premium, Apple-quality warmth. One soft greige canvas, four feeling
/// accents, each with a `*Soft` tint for badges/fills.
class AppColors {
  AppColors._();

  // ── Canvas ────────────────────────────────────────────────
  static const cream = Color(0xFFF2EEEB); // app background (warm greige)
  static const creamDeep = Color(0xFFEAE4DF); // pressed / alt surface
  static const surface = Color(0xFFFAF8F6); // cards
  static const surfaceAlt = Color(0xFFFFFFFF); // pure-white insets

  // ── Gold (primary accent / CTA) ──────────────────────────
  static const peach = Color(0xFFFFD77A);
  static const peachDeep = Color(0xFFE8B84F);
  static const peachSoft = Color(0xFFFBEFD2);

  // ── Sage (success / safe / all-clear) ────────────────────
  static const sage = Color(0xFF7BC47F);
  static const sageDeep = Color(0xFF5BA85F);
  static const sageSoft = Color(0xFFE4F1E5);

  // ── Coral (SOS / alarm) ──────────────────────────────────
  static const coral = Color(0xFFFF6B57);
  static const coralDeep = Color(0xFFE8553F);
  static const coralSoft = Color(0xFFFFE3DD);

  // ── Lavender (Voice SOS / trusted network) ───────────────
  static const lavender = Color(0xFF8B7CF8);
  static const lavenderDeep = Color(0xFF6F5DE0);
  static const lavenderSoft = Color(0xFFECE8FE);

  // ── Gold (mascot / shield) — same family as primary ──────
  static const gold = Color(0xFFFFD77A);
  static const goldDeep = Color(0xFFE8B84F);
  static const goldSoft = Color(0xFFFBEFD2);

  // ── Text ─────────────────────────────────────────────────
  static const textPrimary = Color(0xFF2D2D2D);
  static const textSecondary = Color(0xFF8A837D);
  static const textMuted = Color(0xFFB3ABA4);
  static const textInverse = Color(0xFFFFFFFF);

  // ── Lines (soft, never dark) ─────────────────────────────
  static const border = Color(0x0D2D2D2D); // rgba(45,45,45,0.05)
  static const divider = Color(0x0A2D2D2D); // rgba(45,45,45,0.04)
  static const overlay = Color(0x73282624); // rgba(40,38,36,0.45)

  // ── Semantic ─────────────────────────────────────────────
  static const success = sage;
  static const warning = peach;
  static const error = coral;
}
